docs/source/learn/pytorch-on-xla-devices.md

```diff
@@ -346,6 +346,72 @@ device is unavailable the load will fail. PyTorch/XLA, like all of
 PyTorch, is under active development and this behavior may change in the
 future.
 
+### Tensor Synchronization During Tracing
+
+While tensor synchronization is normal for JIT workflow, it is not expected during traced inference (i.e. traced workflow in AWS Neuron).
+When working with traced graphs, developers may encounter issues related to tensor synchronization during tracing, which can lead to additional graphs being compiled and unexpected program behavior.
```

Comment:
In PyTorch/XLA, we're working with traced graphs all the time. Could you clarify what's meant by "When working with traced graphs"?